### DISCUSSION TITLE
CSS framework?

### DISCUSSION THREAD

**Discussion by ahornace** [oracle/opengrok#3957] [Development]
The UI needs some revamp as it is quite outdated. I think CSS framework would make it easier for us (just engineers without any designers) to provide a decent-looking site. However, which one to use? 

I've used Bootstrap a little and was satisfied with it and it seems to be the most popular option. UIkit seems to be used in OCI docs.

The list of frameworks:
https://github.com/troxler/awesome-css-frameworks

Related issues:
https://github.com/oracle/opengrok/issues/3215
https://github.com/oracle/opengrok/issues/2406

**Comment by ktulinger** [comment]
I can't say a bad word about bootstrap. Some of the existing components I designed (main page groupings) are inspired by bootstrap already.
